js/app.js

```diff
@@ -0,0 +1,487 @@
+
+//audio constants
+const dealCard = new Audio("../audio/cardDeal.ogg")
+dealCard.volume = .2
+const cardSlide = new Audio("../audio/cardSlide.wav")
+cardSlide.volume = .2
+const clap = new Audio("../audio/clap.wav")
+clap.volume = .1
+const tada = new Audio("../audio/tadaG.flac")
+tada.volume = .1
+const no = new Audio("../audio/MichaelNo.mp3")
+no.volume = .1
+const chip = new Audio("../audio/casinoChip.wav")
+chip.volume = .1
+
+/*---------------------------- Variables (state) ----------------------------*/
+let phase
+let score = 2000
+let goal = 30000
+let playerHand = []
+let bet
+let outcome = ""  //⬅️ who is winner of compares
+
+let dealerHand = []
+
+let playerSplitHand = []
+
+let discardDeck = []
+
+let blackjackDeck = [...sixDeckOrig]
+
+let playerTotal = 0
+let dealerTotal = 0
+let valueHolder
+let totalHolder
+let tempArray = []
+
+/*------------------------ Cached Element References ------------------------*/
+let playerMenu = document.getElementById('btn-group')
+let playGame = document.getElementById('phaseGame')
+let scoreArea = document.getElementById('scoreboard')
+let betArea = document.getElementById('betting')
+
+let playerHandElem = document.getElementById('playerHand')
+let dealerHandElem = document.getElementById('dealerHand')
+
+let player1TotalElem = document.getElementById('playerTotal')
+let dealerTotalElem = document.getElementById('dealerTotal')
```

Comment:
You should be using `const` for most of your cached element references to prevent them from re-assignment.